configure-ironic.sh

```diff
@@ -2,59 +2,40 @@
 
 . /bin/ironic-common.sh
 
-HTTP_PORT=${HTTP_PORT:-"80"}
-MARIADB_PASSWORD=${MARIADB_PASSWORD:-"change_me"}
-NUMPROC=$(cat /proc/cpuinfo  | grep "^processor" | wc -l)
-NUMWORKERS=$(( NUMPROC < 12 ? NUMPROC : 12 ))
+export HTTP_PORT=${HTTP_PORT:-"80"}
+export MARIADB_PASSWORD=${MARIADB_PASSWORD:-"change_me"}
+export NUMPROC=$(cat /proc/cpuinfo  | grep "^processor" | wc -l)
+export NUMWORKERS=$(( NUMPROC < 12 ? NUMPROC : 12 ))
 
 # Whether to enable fast_track provisioning or not
-IRONIC_FAST_TRACK=${IRONIC_FAST_TRACK:-true}
+export IRONIC_FAST_TRACK=${IRONIC_FAST_TRACK:-true}
 
 # Whether cleaning disks before and after deployment
-IRONIC_AUTOMATED_CLEAN=${IRONIC_AUTOMATED_CLEAN:-true}
+export IRONIC_AUTOMATED_CLEAN=${IRONIC_AUTOMATED_CLEAN:-true}
 
 wait_for_interface_or_ip
 
-if [[ $IRONIC_FAST_TRACK == true ]]; then
-    INSPECTOR_POWER_OFF=false
-    # TODO(dtantsur): ipa-api-url should be populated by ironic itself, but
-    # it's not yet, so working around here.
-    INSPECTOR_EXTRA_ARGS=" ipa-api-url=http://${IRONIC_URL_HOST}:6385"
+if [ -n "$CERT_FILE" ] && [ -z "$KEY_FILE" ] ; then
+    echo "Missing TLS Certificate key file"
+    exit 1
+fi
+if [ -z "$CERT_FILE" ] && [ -n "$KEY_FILE" ] ; then
+    echo "Missing TLS Certificate file"
+    exit 1
+fi
+
```

Comment:
If `CERT_FILE` is set, but `CACERT_FILE` is not, lets default `CACERT_FILE=$CERT_FILE`. Otherwise the case of not providing a CA won't work.

Reply:
fixed, thank you. I added an export to set that variable in configure-ironic.sh when it is unset